gemma-core/pom.xml

```diff
@@ -117,13 +108,26 @@
                     </dependency>
                 </dependencies>
             </plugin>
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-jar-plugin</artifactId>
+                <version>3.2.0</version>
+                <executions>
+                    <execution>
+                        <goals>
+                            <goal>test-jar</goal>
+                        </goals>
+                    </execution>
+                </executions>
+            </plugin>
             <plugin>
                 <groupId>org.codehaus.mojo</groupId>
                 <artifactId>appassembler-maven-plugin</artifactId>
                 <version>2.1.0</version>
                 <configuration>
                     <repositoryName>lib</repositoryName>
                     <repositoryLayout>flat</repositoryLayout>
+                    <extraJvmArguments>-Dlog4j.configurationFile=@BASEDIR@/etc/log4j.properties</extraJvmArguments>
```

Comment:
It might be worth to document that somewhere. In an installed CLI distribution, the log4j.properties can be overriden under `etc/log4j.properties`. I already updated this on our internal setup, so the changes will be effective for this release.